var/spack/repos/builtin/packages/py-platformdirs/package.py

```diff
@@ -22,6 +22,8 @@ class PyPlatformdirs(PythonPackage):
     depends_on("py-setuptools@44:", type="build")
     depends_on("py-setuptools-scm@5:+toml", type="build")
 
+
+class PythonPipBuilder(spack.build_systems.python.PythonPipBuilder):
```

Comment:
Would it be possible to include `*Builder` in `spack.package.__all__`? That would help simplify these packages

Reply:
No. The selection of builders right now is based on names, which has the benefit of prescribing names for the users so that they don't have to come up with one themselves + use other registration procedures. For instance, a `PythonPipBuilder` is the builder associated with the `python_pip` build-system. If we include the names in `spack.package.__all__` there will be a name clash.

That said I agree that the line to type is long, and to help users I updated `spack create` to conform to the new style.

Reply:
Hmm, is it a name clash? I would assume Python would overwrite the variable to the new name in a situation like:
```python
from spack.build_systems.python import PythonPipBuilder

class PythonPipBuilder(PythonPipBuilder):
    ...
```

Reply:
Let me be clear, I don't mean a `SyntaxError`, but the name `PythonPipBuilder` would refer to different objects in the same file depending on the context.

Reply:
Is that an issue? Will users need/want to reference `PythonPipBuilder` in any context other than defining a new subclass?

Reply:
They might reference it in a class context, so I'd like it keeps the same meaning throughout the file.

Reply:
What I can try to do here is to add an alias to the `*Package`:
```python
class CMakeBuilder(CMakePackage.DefaultBuilder):
    pass
```